newscript.go

```
@@ -80,20 +77,42 @@ func setup() error {
 	if err := cmd.Run(); err != nil {
 		return err
 	}
-	/*err = blankBootstick()
+	err = blankBootstick()
 	if err != nil {
 		return err
-	}*/
+	}
 	return nil
 
 }
 
+func deviceAndLocation() {
+	cmd := exec.Command("lsblk")
+	cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
+	if err := cmd.Run(); err != nil {
+		log.Print("unable to print output of lsblk")
```

you should print the error here but really you should also return the error.